### DISCUSSION TITLE
JSON URL

### DISCUSSION THREAD

**Discussion by stphdee** [gapitio/gapit-htmlgraphics-panel#105] [Q&A] [answered]
Hi. How can I use JSON URL to get data and display data on the panel? Thank you.

**Comment by ZuperZee** [comment, marked as answer]
Could you explain a little more?

Are you asking about how to get data from [JSON API plugin](https://grafana.com/grafana/plugins/marcusolsson-json-datasource/)?

**Reply by stphdee** [reply]
Hi. I used Infinity plugin for my data source (using JSON) and I want to use the HTML Graphics plugin to display my data in the panel. How can I get and set my data from the data source to HTML Graphics so that it will display in the panel?

**Reply by ZuperZee** [reply]
I haven't used that data source plugin before, but I imagine it should be similar to how the other data sources work.
You could try to log [data](https://gapit-htmlgraphics-panel.gapit.io/docs/references/#data-global) (`console.log(data)`) and see what your `data.series` response is. If the default panel doesn't show anything when a query has been added, the data object might be slightly different (like not having two fields for time and value, but only a single value field).

**Reply by stphdee** [reply]
I can now get data from the data source. Will working on display the data to the panel. Thanks a lot.
